Doc/c-api/sys.rst

```diff
@@ -240,11 +240,6 @@ Operating System Utilities
 
    Call ``fclose(file)``.
```

Comment:
I know it's mentioned in the `Py_fopen` docs, but for people who just read this one, it's worth adding a bit more clarity.

```suggestion
   Closes files that were opened by :c:func:`Py_fopen`.
```

Reply:
> (Probably should also explain the return value... it's just errno or 0, right?)

Done.